### Actividad 5

#### Caso 1 
**Se sabe que la media del consumo de energía eléctrica en cierta provincia es de 721 kwh. Una empresa tecnológica de la región cree que sus empleados consumen más que  el  promedio provincial. Recoge  información  sobre  los  consumos  de  20 empleados escogidos al azar, y obtiene los siguientes datos:**

  **Si la distribución del consumo mensual de energía eléctrica es normal:**
    


In [3]:
import numpy as np
from scipy import stats

In [4]:
sample = np.array(
    [710,774,814,768,823,
     732,675,755,770,660,
     654,757,736,677,797,
     760,718,774,747,796]
    )

- a)  ¿Hay  evidencias  para  afirmar que  el  promedio del consumo de  energía eléctrica hogareño de los empleados de la empresa es superior a la media del consumo a nivel provincial? Usar un nivel de significación del 10%.

**X**: Consumo mensual de energia electrica de un empleado.
Sabemos que la distribucion es normal pero desconocemos la media muestral ($\mu$) y desviacion ($\sigma$).

Planteamos la hipotesis cero:

$H_0 : \mu = 721$

eh hipotesis alternativa:

$H_1 : \mu > 721$

Como no se conoce la desviacion estandar poblacional se debe estimar con la desviacion estandar muestral $S$.

$\frac{\overline{X}-\mu}{S/\sqrt{n}} \sim t_{n-1} $

El nivel de significancia es del 10%

Como la prueba es unilateral a la derecha, se rechazara la hipotesis nula si el estadistico es muy grande.

$t_{19;0,9} = 1.3277$

Rechazaremos $H_0$ si el estadistico de prueba es mayor o igual a 1.3277
No rechazaremos $H_0$ si el estadistico de prueba es menor que 1.377 

In [18]:
data_mean = sample.mean()
data_n = sample.shape[0]
data_std = sample.std(ddof=1)

print(f"media: {data_mean:.0f}\ndesviacion estandar muestral: {data_std:.2f}")

media: 745
desviacion estandar muestral: 49.57


- b)  ¿Cuál es el valor p de la decisión?

In [35]:
from scipy.stats import norm

In [45]:
p_value = 1-norm.cdf((data_mean-721)/(data_std/np.sqrt(data_n)))
p_value*100

1.5715441752677695

Se rechaza la hipotesis nula y este valor de p-value quiere decir que existe un 1.6% de que este nivel de consumo haya sido al azar por tanto es probable que este sea el consumo real de los empleados en esa provincia.